analyses/cell-type-ewings/scripts/make-gene-order-file.R

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env Rscript
+
+project_root <- here::here()
+renv::load(project_root)
+
+library(optparse)
+
+option_list <- list(
+  make_option(
+    opt_str = c("--gtf_file"),
+    type = "character",
+    default = "s3://scpca-references/homo_sapiens/ensembl-104/annotation/Homo_sapiens.GRCh38.104.gtf.gz",
+    help = "URI to gtf file to create gene order file."
+  ),
+  make_option(
+    opt_str = c("--local_ref_dir"),
+    type = "character",
+    default = file.path(project_root, "references", "infercnv_refs"), 
+    help = "Directory to use for saving GTF file and gene order file."
+  )
+)
+
+# Parse options
+opt <- parse_args(OptionParser(option_list = option_list))
+
+# create ref directory if doesn't already exist 
+fs::dir_create(opt$local_ref_dir)
+
+# sync gtf file to local directory 
+gtf_filename <- stringr::word(opt$gtf_file, -1, sep = "/")
```

Comment:
simpler and probably more robust
```suggestion
gtf_filename <- basename(opt$gtf_file)
```